testing/vips/APKBUILD

```diff
@@ -1,21 +1,19 @@
 # Contributor: Will Jordan <will.jordan@gmail.com>
 # Maintainer: Will Jordan <will.jordan@gmail.com>
 pkgname=vips
-pkgver=8.5.6
-pkgrel=1
+pkgver=8.5.8
+pkgrel=0
 pkgdesc="A fast image processing library with low memory needs"
 url="http://www.vips.ecs.soton.ac.uk/"
 arch="all"
 license="LGPL 2.1+"
-depends=""
 depends_dev="libjpeg-turbo-dev libexif-dev lcms2-dev"
 makedepends="$depends_dev fftw-dev giflib-dev glib-dev libpng-dev libwebp-dev
 	expat-dev orc-dev tiff-dev"
-install=""
 subpackages="$pkgname-dev $pkgname-doc $pkgname-lang $pkgname-tools"
-source="https://github.com/jcupitt/libvips/releases/download/v$pkgver/vips-$pkgver.tar.gz"
+source="https://github.com/jcupitt/libvips/releases/download/v$pkgver/$pkgname-$pkgver.tar.gz"
 
-builddir="$srcdir"/vips-$pkgver
+builddir="$srcdir/$pkgname-$pkgver"
```

Comment:
its a matter of taste, but I prefer to avoid using `$pkgname` in `builddir` and `source` because I think it provides no value and it makes it harder to rename the package if we ever do an `vips2` package or similar.